src/components/MARCTable/MARCTable.test.js

```diff
@@ -0,0 +1,352 @@
+import React from 'react';
+import {
+  render,
+  cleanup,
```

Comment:
The same point about cleanup.

https://testing-library.com/docs/react-testing-library/setup#skipping-auto-cleanup